unblob/finder.py

```diff
@@ -98,13 +110,29 @@ def search_chunks_by_priority(  # noqa: C901
                     continue
 
                 if chunk.start_offset < 0 or chunk.end_offset < 0:
-                    exit_code_var.set(1)
-                    logger.error("Chunk has negative offset", chunk=chunk)
+                    error_text = "Chunk has negative offset"
+                    error_queue.put(
```

Comment:
Naming of the same thing should be consistent throughout the codebase. `report_queue` would be the most generic IMO.